src/extension/commands_helpers.c

```diff
@@ -104,10 +104,23 @@ static dd_result _dd_command_exec(dd_conn *nonnull conn, bool check_cred,
             return res;
         }
 
-        res = spec->incoming_cb(imsg.root, ctx);
+        mpack_node_t first_response = mpack_node_array_at(imsg.root, 0);
+        mpack_error_t err = mpack_node_error(first_response);
+        if (err != mpack_ok) {
+            mlog(dd_log_error, "Array of responses could not be retrieved - %s",
+                mpack_error_to_string(err));
+        }
+        mpack_node_t first_message = mpack_node_array_at(first_response, 1);
```

Comment:
What is this? Is it because now the message is `[<type>, <message>]`?

Reply:
Yes

Reply:
For now the extension is not paying attention to the type